K8s-deployment/Charts/consent-validator/example-secrets/config.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "1.0",
+    "postgresOptions": {
+        "databaseIP": "database.ip",
+        "databasePort": "5433",
+        "databaseName": "db-name",
+        "databaseSchema": "db-schema",
+        "databaseUserName": "db-user",
+        "databasePassword": "db-password",
```

Comment:
add shell command to retrieve the password. Refer to sample configs of other API servers.